### PR TITLE
[MM-61]: Added the test cases for the PR 683 and for PR 718 .

### PR DESCRIPTION
**Pull request by arush-vashishtha**
### Summary
This PR consist the test cases for the following PRs,
https://github.com/mattermost/mattermost-plugin-github/pull/718
https://github.com/mattermost/mattermost-plugin-github/pull/683

### CODE REVIEW

File: data/test-cases/plugins/github/subscriptions/Exclude_subscriptions_for_repo.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Enter the slash command `/github subscriptions add <organization> --feature <events> --exclude <repository>` in the desired channel or DM/GM on MM and create subscription for a organization with more than one repository and exclude any desired repository.
+2. Create the desired event in the desired exclude repository on the Github and navigate to the desired channel or DM/GM on MM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/github/subscriptions/Exclude_subscriptions_for_repo.md]: Rephrase this.

File: data/test-cases/plugins/github/subscriptions/Exclude_subscriptions_for_repo.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**Step 2**
+
+1. Enter the slash command `/github subscriptions add <organization> --feature <events> --exclude <repository>` in the desired channel or DM/GM on MM and create subscription for a organization with more than one repository and exclude any desired repository.
+2. Create the desired event in any desired repository except the excluded ones on the Github and navigate to the desired channel or DM/GM on MM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/github/subscriptions/Exclude_subscriptions_for_repo.md]: Same as above

**Review by AayushChaudhary0001 — APPROVED** [review]

File: data/test-cases/plugins/github/subscriptions/Re-opened_PR_notification.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# (Required) Ensure all values are filled up
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/github/subscriptions/Re-opened_PR_notification.md:2]:
```suggestion
---
# (Required) Ensure all values are filled up
```

File: data/test-cases/plugins/github/subscriptions/Exclude_subscriptions_for_repo.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# (Required) Ensure all values are filled up
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/github/subscriptions/Exclude_subscriptions_for_repo.md:2]:
```suggestion
---
# (Required) Ensure all values are filled up
```

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

**Review by raghavaggarwal2308 — APPROVED** [review]

**Pull request merged**